splicemachine/spark/context.py

```diff
@@ -100,26 +123,83 @@ def dropTable(self, schema_table_name):  # works
         """
         return self.context.dropTable(schema_table_name)
 
+    def dropTable2(self, schema_name, table_name):
```

Comment:
I think we should combine these two functions so users can pass in either to a single dropTable

Reply:
It might be good to combine them.  Thinking about the parameter list, would it be like:
`def dropTable(self, param1, param2=None)`
so they could call dropTable('schema.table') or dropTable('schema', 'table') .  The param names aren't descriptive, but the comments could describe them.  Or maybe another way?

Reply:
I think that idea is correct, and just call the params 
`def dropTable(self, schema_and_or_table_name, table_name=None)`

And then add some logic to check for the `.` in the first param and check if the second is 
None

Reply:
Updated the function in new commit [6ba47bc](https://github.com/splicemachine/pysplice/pull/54/commits/6ba47bcab47845eff66334f14949069862aa25ac).